amundsen_application/static/js/components/common/SearchBar/styles.scss

```diff
@@ -43,19 +43,6 @@
         height: 36px;
         padding: 6px 6px 6px 36px;
       }
-
-      &:focus {
```

Comment:
How come these are removed?

Reply:
The new mocks do not include a focus state and I also recall Matt mentioning that as well.

Reply:
Check the SearchBar in Abstract in the component library. There is an "active" state specified and that is what this code was added for.

Reply:
Oh I see. I thought this was the purple highlight.